docs/start/basics.ipynb

```diff
@@ -680,7 +680,7 @@
    "source": [
     "## Next steps\n",
     "\n",
-    "After completing this tutorial, you should be able to use gates and operations to construct your own quantum circuits, simulate them, and to use sweeps.  It should give you a brief idea of the commonly used \n",
+    "After completing this tutorial, you should be able to use gates and operations to construct your own quantum circuits, simulate them, and to use sweeps.\n",
```

Comment:
Suggestion: use "to" (as in "to do") consistently in each item on the list:

"to construct your own quantum circuits, to simulate them, and to use sweeps."

(Adds "to" in front of "simulate".)